plugins/python-build/bin/python-build

```diff
@@ -1010,7 +1010,7 @@ build_package_micropython() {
   fi
   { cd mpy-cross
     "$MAKE" $MAKE_OPTS
```

Comment:
wouldn't it be simpler if you do as "$MAKE" -C mpy-cross $MAKE_OPTS ...?

Reply:
> wouldn't it be simpler if you do as `"$MAKE" -C mpy-cross $MAKE_OPTS`

yes, you are totally right, feel free to make the change as you suggested

Reply:
Let's defer the change for now until something else comes up 😉